helmfile/charts/gatekeeper-metrics/templates/gatekeeper-audit-metrics.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: gatekeeper-metrics
+    chart: gatekeeper-metrics
+    control-plane: audit-controller
+    type: metrics
+  name: gatekeeper-audit-metrics
+  namespace: gatekeeper-system
```

Comment:
Skip the namespaces if possible so they come from where we install the chart instead of being hard coded.
```suggestion
```